packages/core/components/QueryPart/QueryPartRow.module.css

```diff
@@ -32,19 +50,25 @@
 
 .row-title {
     display: flex;
-    width: calc(100% - 33px);
+    max-width: calc(100% - 33px);
+    width: 100%;
```

Comment:
Some vertical misalignment for title vs icon. May be the same issue as the comment I put for filter
<img width="317" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/40ff5a37-e87a-4ae6-8c0b-e2628746809c">

Reply:
Fixed